packages/react-aria-components/docs/Autocomplete.mdx

```diff
@@ -108,14 +119,301 @@ function Example() {
 `Autocomplete` is a controller for a child text input, such as a [TextField](TextField.html) or [SearchField](SearchField.html), and a collection component such as a [Menu](Menu.html) or [ListBox](ListBox.html). It enables the user to filter a list of items, and navigate via the arrow keys while keeping focus on the input.
 
 ```tsx
-import {UNSTABLE_Autocomplete as Autocomplete, SearchField, Menu} from 'react-aria-components';
+import {Autocomplete, SearchField, Menu} from 'react-aria-components';
 
 <Autocomplete>
+  <SearchField />   {/*   or <TextField />   */}
+  <Menu />   {/*   or <ListBox />   */}
+</Autocomplete>
+```
+
```

Comment:
I think we need to add `outline: none` by default in SearchField.mdx to prevent the browser default focus ring from showing up (except when `[data-focused]`)?